src/docfx/lib/log/Telemetry.cs

```diff
@@ -17,47 +18,75 @@ internal static class Telemetry
     // https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/EventData.bond#L19
     private const int MaxEventPropertyLength = 8192;
     private const int MaxChildrenLength = 5;
-    private static readonly TelemetryClient s_telemetryClient = new(TelemetryConfiguration.CreateDefault());
+
+    private static TelemetryClient TelemetryClient { get; }
```

Comment:
Why is this turned into properties and not inline initialized as before? If the `TelemetryClient` initialization cannot be done with a single expression, you could use a helper method:
```suggestion
    private static readonly TelemetryClient s_telemetryClient = CreateTelemetryClient();
```

Reply:
Updated